### DISCUSSION TITLE
Adding cartesian coordinate widget placement support and editors for propgriditem

### DISCUSSION THREAD

**Discussion by johnf1nk1337** [wxFormBuilder/wxFormBuilder#751] [Ideas]
Would it be possible to add the support for cartesian coordinate widget placement as well as editors in the properties grid for PropGridItem into the next release so people can build uis without sizers and apply initial values to the propgriditems. If there is a way to do this already and I just don't know about it could you please inform me as to how to do this. 

Thank You

**Comment by sodevel** [comment]
Are you talking about absolute positioning without sizers? wxFB is highly designed to work with sizers, i'm afraid it is too much work to change this and i also think it is not of much use these days. I really want to enable the use of dynamic size units like Dialog Units and DIPs to finally deal with HiDPI layouts, but it's still a long way to reach this goal.

About the Property Grid enhancement, i am not familiar with this control and can't judge how difficult it is to support this. Since i am short on time and have other tasks to complete, i won't be able to implement this. PRs are welcome.
